utils/config.go

```diff
@@ -387,6 +391,20 @@ func ValidateLdapFilter(cfg *model.Config) *model.AppError {
 	return nil
 }
 
+func ValidateLocales(cfg *model.Config) *model.AppError {
+	if len(*cfg.LocalizationSettings.AvailableLocales) > 0 {
+		for _, word := range strings.Split(*cfg.LocalizationSettings.AvailableLocales, ",") {
+			if word == *cfg.LocalizationSettings.DefaultClientLocale {
+				return nil
+			}
+		}
+
+		return model.NewLocAppError("ValidateLocales", "utils.config.validate_locale.app_error", nil, "")
+	}
+
+	return nil
```

Comment:
Yes, exactly. And then you will start to see problems with new users. Follow these steps to reproduce:

1. Set `DefaultClientLocale` to `fi` (finnish)
2. Make sure `AvailableLocales` is empty: ""
3. Register a new user trough the invitation link
4. Log in as a new user and go to Account Settings -> Display
5. Notice that there is a Javascript error and huge stack traces in the console.

The error you see is the same one as in PLT-5548.